Directory.Build.targets

```diff
@@ -3,4 +3,18 @@
   <Target Name="VSTestIfTestProject">
     <CallTarget Targets="VSTest" Condition="'$(IsTestProject)' == 'true'" />
   </Target>
+  <Target Name="ComputePackageVersion" AfterTargets="MinVer" Condition="'$(BUILD_PR)' != ''" >
+    <PropertyGroup>
+      <PackageVersion>$(MinVerMajor).$(MinVerMinor).$(MinVerPatch)-pr.$(BUILD_PR)</PackageVersion>
```

Comment:
FTR this will also trigger NU1505 because of the dot after `pr`. Not that it matters (I don't belive it's worth supporting those older clients) but it's another instance where it's not just height that triggers it. It may be worth altering the comment before the suppression of NU1505.

Reply:
Good point, will do (elsewhere)